tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -618,3 +618,74 @@ Feature: Sharing files and folders with internal users
       | uid_owner  | user1      |
       | share_with | user2      |
       | expiration | +7         |
+
+  Scenario: share a resource with another internal user with expiration date within enforced maximum expiration date
+    Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
+    And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
+    And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "5"
+    And user "user1" has logged in using the webUI
+    When the user shares file "lorem.txt" with user "User Two" which expires in "+4" days using the webUI
+    Then user "user1" should have a share with these details:
+      | field      | value      |
+      | path       | /lorem.txt |
+      | share_type | user       |
+      | uid_owner  | user1      |
+      | share_with | user2      |
+      | expiration | +4         |
+
+  Scenario Outline: share a resource with another internal user with expiration date beyond enforced maximum expiration date
+    Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"
+    And the setting "shareapi_enforce_expire_date_user_share" of app "core" has been set to "yes"
+    And the setting "shareapi_expire_after_n_days_user_share" of app "core" has been set to "5"
+    And user "user1" has logged in using the webUI
+    When the user tries to share file "<shared-resource>" with user "User Two" which expires in "+6" days using the webUI
```

Comment:
```suggestion
    When the user tries to share the resource "<shared-resource>" with user "User Two" which expires in "+6" days using the webUI
```
should not say file, because one of the examples is a folder